omod/src/main/web/dashboardwidgets/obsacrossencounters/obsacrossencounters.controller.js

```diff
@@ -92,6 +101,13 @@ export default class ObsAcrossEncountersController {
                     //If value is a concept
                     obs.value = obs.value.display;
                 }
+                else if (['8d4a505e-c2cc-11de-8d13-0010c6dffd0f',
+                        '8d4a591e-c2cc-11de-8d13-0010c6dffd0f',
+                        '8d4a5af4-c2cc-11de-8d13-0010c6dffd0f'].indexOf(obs.concept.datatype.uuid) > -1) {
```

Comment:
Minor point, but I would define this list of three uuids as a constant at the top o f the file and name it something like "DATE_AND_TIME_DATATYPE_UUIDS", then it will be more clear what is happening here.

Reply:
@mogoodrich On your answer of the concept data type, I am not sure which concept is being checked with this line of code, whether its just the main concept on the encounter or the associated concept - I suspect the main concept on the encounter

Reply:
You mean the main concept on the obs?  Not sure what main concept on the encounter means?

Reply:
@mogoodrich this widget loads selected concepts from encounters that match provided uuid - those are the main concept.

Reply:
@suthagar23 You need the data type in the custom representation for for the concept in obs since the check is for obs.concept.datatype (other widgets use full representation fto his)

Reply:
@suthagar23 Please ignore looks like the mistake on my end - I was looking at the obs data not the concept one